internal/schemautil/mutations.go

```diff
@@ -0,0 +1,80 @@
+package schemautil
+
+// sensitiveFields is a list of fields that are not returned by the API on a refresh, but are supposed to remain in the
+// state to make Terraform work properly.
+var sensitiveFields = []string{
+	"admin_username",
+	"admin_password",
+}
+
+// copySensitiveFields preserves sensitive fields in the state that are not returned by the API on a refresh.
+func copySensitiveFields(old, new map[string]interface{}) {
+	for _, k := range sensitiveFields {
+		if v, ok := old[k]; ok {
+			new[k] = v
+		}
+	}
+}
+
+// normalizeIpFilter compares a list of IP filters set in the oldUserConfig and a sorted version coming from
+// newUserConfig and returns the re-sorted IP filters, such that all matching entries will be in
+// the same order as defined in the oldUserConfig.
+func normalizeIpFilter(old, new map[string]interface{}) {
+	oldIpFilters, ok := old["ip_filter"].([]interface{})
+	if !ok {
+		return
+	}
+
+	newIpFilters, ok := new["ip_filter"].([]interface{})
+	if !ok {
+		return
+	}
+
+	var normalizedIpFilters []interface{}
+
+	// TODO: ipFilterDiff allows for ip_filter field to be managed via Aiven Console aka Web UI.
```

Comment:
Can you please explain manning of this TODO, and how this is related to TF project?

Reply:
the sorting logic that stores `ip_filter` entries in `ipFilterDiff` is only needed if we want to allow editing them via Aiven Console, or other API clients